package.json

```diff
@@ -128,4 +128,4 @@
     "ts-node": "^10.7.0",
     "typescript": "~4.4.4"
   }
-}
+}
```

Comment:
```suggestion
}

```
Again new line (whenever you see 👇🏽 )
![Screenshot from 2023-01-25 17-07-31](https://user-images.githubusercontent.com/1779590/214584864-b01ab464-8d8e-4aa2-958b-f01642dbeb26.png)